tests/manage/z_cluster/test_change_policy_scale_up_down.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants, node
+from ocs_ci.helpers.helpers import create_pod, wait_for_resource_state
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@bugzilla("2024870")
+class TestChangePolicyScaleUpDown(ManageTest):
```

Comment:
where does scale up and down take place in the test?

Reply:
I have an issue with `dd` cmd